llvm/include/llvm/MC/MCParser/MCTargetAsmParser.h

```diff
@@ -439,14 +439,14 @@ class MCTargetAsmParser : public MCAsmParserExtension {
   /// \param Operands [out] - The list of parsed operands, this returns
   ///        ownership of them to the caller.
   /// \return True on failure.
-  virtual bool ParseInstruction(ParseInstructionInfo &Info, StringRef Name,
+  virtual bool parseInstruction(ParseInstructionInfo &Info, StringRef Name,
                                 SMLoc NameLoc, OperandVector &Operands) = 0;
-  virtual bool ParseInstruction(ParseInstructionInfo &Info, StringRef Name,
+  virtual bool parseInstruction(ParseInstructionInfo &Info, StringRef Name,
                                 AsmToken Token, OperandVector &Operands) {
-    return ParseInstruction(Info, Name, Token.getLoc(), Operands);
+    return parseInstruction(Info, Name, Token.getLoc(), Operands);
   }
 
-  /// ParseDirective - Parse a target specific assembler directive
+  /// parseDirectives - Parse a target specific assembler directive
```

Comment:
Per coding standard, the comment shouldn't include function name.
```suggestion
  /// Parses a target specific assembler directive.
```